src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/service/SignedOperationCapacityService.kt

```diff
@@ -24,18 +28,18 @@ class SignedOperationCapacityService(
   }
 
   @Transactional
-  fun saveSignedOperationalCapacity(prisonId: String, oc: PrisonSignedOperationCapacity) {
-    // TODO need to be changed to dto
-    val opdb = prisonSignedOperationalCapacityRepository.findOneByPrisonId(prisonId) ?: PrisonSignedOperationCapacity(
-      signedOperationCapacity = oc.signedOperationCapacity,
-      prisonId = prisonId,
-      dateTime = oc.dateTime,
-      updatedBy = oc.updatedBy,
+  fun saveSignedOperationalCapacity(request: SignedOperationCapacityValidRequest): PrisonSignedOperationCapacity {
+
+    val opdb = prisonSignedOperationalCapacityRepository.findOneByPrisonId(request.prisonId) ?: PrisonSignedOperationCapacity(
```

Comment:
I think DTO can create the request as LocationDTO and give the LocalDateTime as well